Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -63,6 +63,24 @@ PaymentMethodsViewModelInputs, PaymentMethodsViewModelOutputs {
     self.goToAddCardScreen = self.didTapAddCardButtonProperty.signal
 
     self.tableViewIsEditing = self.editButtonTappedSignal.scan(false) { current, _ in !current }
+
+    // Koala:
+    self.viewDidLoadProperty.signal
```

Comment:
Wondering if this event should be on `viewDidAppear` 🤔

Reply:
Many of our `Viewed` tracking events are triggered on `viewDidLoad`. I could confirm on Looker that the events are being sent, though. 
The difference is that on `viewDidLoad` the event will only be triggered once, while on `viewDidAppear` it can be triggered more than once, for example once when we open the view and again case we dismiss a child view controller, if any. I wonder which case we would consider as ideal 🤔

Reply:
From conversation with @terminalbae the expectation is that we track this each time the user navigates to the view. So `viewDidAppear` is the correct event. I think that there are some occasions where we use `viewWillAppear` and we actually filter based on `animated` being `false` - [example](https://github.com/kickstarter/ios-oss/blob/master/Library/ViewModels/ActivitiesViewModel.swift#L270). I don't know for certain as there doesn't seem to be documentation on this, but I would guess that this is being done because of view controller containment and that when a VC appears in a child container then `animated` is `false` and that we're tracking that specifically.

I should probably just test this to be sure - so let me see if I can do that real quick.

Reply:
Thank you for your investigation. So I will change to `viewWillAppear` 👍

Reply:
If we track this in `viewWillAppear` do we want to filter on `animated` being `false`?

Reply:
For events being tracked on `viewWillAppear` will only be filtered on `animated` being false is they are `RootTabBarViewController`'s child (such as Discovery, Activity, Dashboard...), this is because we are setting the value manually to false while configuring the tabBar's viewcontrollers (you can see it on `RootTabBarViewController.swift`)

Reply:
Looks like `setViewControllers($0, animated: false)` doesn't affect `viewWillAppear` at all.